scripts/run-local-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+ACCOUNT_ID=XXX
+ACCESS_TOKEN=XXX
+USERNAME=betademo
+PASSWORD=nottobes
+
+export ACCOUNT_ID
+export ACCESS_TOKEN
+export USERNAME
+export PASSWORD
+
+# shellcheck disable=SC2046
+go test -v $(go list ./... | grep -v '/vendor/' | grep -v '/integration_tests')
+ginkgo -focus='Broker with fake Compose client' integration_tests/
```

Comment:
👍 We'll also create a helper to reduce code duplication. We'll rename packages to `fake_api_integrations_tests` and `real_api_integration_tests`.